lib/landing_session.js

```diff
@@ -155,14 +161,28 @@ class LandingSession extends Session {
         return;
       }
       return this.final();
+    } else if (this.autorebase && this.canAutomaticallyRebase(subjects)) {
+      // Run git rebase in interactive mode with autosquash but without editor
+      // so that it will perform everything automatically.
+      cli.log(`There are ${subjects.length} commits in the PR. ` +
+        'Attempting autorebase.');
+      const { upstream, branch } = this;
+      const msgAmend = '-x "git-node land --amend"';
```

Comment:
This fixes the interaction with --yes:

```suggestion
      const msgAmend = 
          `-x "git-node land --amend ${cli.assumeYes ? '--yes' : ''}"`;
```

Reply:
Oh, that's where it was, thanks.